src/main/java/com/example/DiceGameBE/service/impl/DiceService.java

```diff
@@ -0,0 +1,38 @@
+package com.example.DiceGameBE.service.impl;
+
+import com.example.DiceGameBE.model.Dice;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.stream.IntStream;
+
+@Service
+@Slf4j
+@RequiredArgsConstructor
+public class DiceService {
+
+    public List<Dice> rollDice() {
```

Comment:
Zgodnie z założeniami gry, metoda rollDice powinna być bardziej generyczna, nie zawsze będzie to rzut 5 kostkami, czasem będzie to rzut 1/2/3 lub 4. Uwzględnij to w parametrze metody